LondonBritishLibrary/orient/BLorient604.xml

```diff
@@ -33,43 +34,234 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+    <title xml:lang="en" type="complete" ref="LIT3998Dersan">Discourse of <persName ref="PRS3251Cyriacus">Cyriacus of Bǝhnesā</persName></title>
+    <incipit>በእንተ፡ ክብራ፡ ለድንግል፡ ንጽሕት፡ ማርያም፡</incipit>
+    <explicit type="subscription" xml:lang="gez">ለዛቲ፡ መጽሐፍ፡ ዘተርጐማ፡ ርቱዓ፡ ሃይማኖት፡ አቡነ፡ አባ፡ ሰላማ፡ ወይጽሕፍ፡ እግዚአብሔር፡ ስሞ<gap reason="omitted" resp="bm:Wright1877BM"/></explicit>
+    <textLang mainLang="gez"></textLang>
+    </msItem>
+
+
+
+
+
                     </msContents>
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
-                                    
+
                                 </support>
                                 <extent>
-                                    <measure unit="leaf"/>
+                                    <measure unit="leaf">137</measure>
                                     <dimensions type="outer" unit="in">
-                                        <height/>
-                                        <width/>
+                                        <height>14.125</height>
+                                        <width>12.5</width>
                                     </dimensions>
                                 </extent>
                             </supportDesc>
                             <layoutDesc>
                                 <layout columns="3" writtenLines="28"/>
-                                
+
                             </layoutDesc>
                         </objectDesc>
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic"/>
-                            
+
                         </handDesc>
-                        
+<additions>
+<list>
+
+<item xml:id="a1">
+<desc type="MixedNote">The volume is full of lists of furniture, dress and books, deeds of gift, and similar documents, in Amharic, written
+on the blank pages and on the margins. They seem to belong chiefly to the reigns of <persName ref="PRS10473Yostos">Yosṭos</persName>,
+<persName ref="PRS5617IyasuII">ʾIyāsu II</persName>, <persName ref="PRS5636IyoasI">ʾIyoʾās I</persName>,
+<persName ref="PRS10376Yohanne">John II</persName>, <persName ref="PRS9152TaklaHa">Takla Hāymānot</persName>, <persName ref="PRS8421Salomon">Solomon</persName>, <persName ref="PRS9142TaklaGi">Takla Giyorgis</persName>, <persName ref="PRS5618IyasuII">Iyāsu III</persName>,
+<persName ref="PRS5332hezqeyas">Ḥǝzqǝyās</persName>, <persName ref="PRS3501Demetros">Dǝmeṭros</persName>, <persName ref="PRS3692egwalas">ʾƎgʷāla Ṣǝyon</persName>,
+ʾIyoʾās II, <persName ref="PRS4661Gigar">Gigār</persName>, <persName ref="PRS5619IyasuIV">Iyāsu IV</persName>, <persName ref="PRS4264GabraKr">Gabra Krǝstos</persName>, <persName ref="PRS8344sahlaDe">Śāhla Dǝngǝl</persName> and
+<persName ref="PRS9430Tewodros">Theodore</persName>.</desc>
+</item>
```

Comment:
Please, add an empty q with the language

Reply:
Is this what you meant?

Reply:
yes, but it should be am in accordance the description provided